Source/Fuse.PushNotifications/iOS/AppDelegatePushNotify.mm

```diff
@@ -9,17 +9,9 @@ - (void)application:(UIApplication *)application initializePushNotifications:(NS
 	if (launchOptions[UIApplicationLaunchOptionsRemoteNotificationKey]) {
 		[self application:application dispatchPushNotification:launchOptions[UIApplicationLaunchOptionsRemoteNotificationKey] fromBar:YES];
 	}
-	if ([application respondsToSelector:@selector(registerUserNotificationSettings:)]) {
-		// use registerUserNotificationSettings
-		[application registerUserNotificationSettings: [UIUserNotificationSettings settingsForTypes:(UIUserNotificationTypeSound  | UIUserNotificationTypeAlert | UIUserNotificationTypeBadge)  categories:nil]];
-		[application registerForRemoteNotifications];
-	} else {
-		// use registerForRemoteNotificationTypes:
-		[application registerForRemoteNotificationTypes:
-		 UIRemoteNotificationTypeBadge |
-		 UIRemoteNotificationTypeSound |
-		 UIRemoteNotificationTypeAlert];
-	}
+#if (!@(Project.iOS.PushNotifications.RegisterOnLaunch:IsSet)) || @(Project.iOS.PushNotifications.RegisterOnLaunch:Or(0))
```

Comment:
This seems really odd to have fuselibs code statically differ based on user project settings. Do we do this anywhere else?  Could this not be a run-time check instead somehow?